### PR TITLE
Add support for RFC 4683

### PR DESCRIPTION
**Pull request by russhousley**
Add module and test for RFC 4683

### CODE REVIEW

File: pyasn1_modules/rfc4683.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# License: http://snmplabs.com/pyasn1/license.html
+#
+# Subject Identification Method (SIM)
+#
+# ASN.1 source from:
+# https://www.rfc-editor.org/rfc/rfc4683.txt
+# https://www.rfc-editor.org/errata/eid1047
+#
+
+from pyasn1.type import univ, char, namedtype, namedval, tag, constraint, useful
```
Comment (etingof) [inline pyasn1_modules/rfc4683.py]: nit: pep8 suggests importing modules one per line

**Comment by etingof** [comment]
Thank you!

File: pyasn1_modules/rfc4683.py (hunk 1 above)
Comment (russhousley) [reply pyasn1_modules/rfc4683.py]: Ooops.  I'll fix this one right now.

Comment (etingof) [reply pyasn1_modules/rfc4683.py]: > Ooops. I'll fix this one right now.

Perhaps in another PR then - this one has been merged.

Comment (russhousley) [reply pyasn1_modules/rfc4683.py]: Okay.  I'll do another PR.

**Pull request merged**